docs/index.rst

```diff
@@ -78,14 +78,15 @@ and secret key configured on your system. You may also specify when
 creating the filesystem instance. Here's how you would do that with an
 opener::
 
-    s3fs = open_fs('s3://<access key>:<secret key>@mybucket')
```

Comment:
This is probably the wrong section, as this covers authentication.

I'd probably add a section under [Opening an S3 Filesystem](http://fs-s3fs.readthedocs.io/en/latest/#opening-an-s3-filesystem) regarding an alternative endpoint.